packages/css/src/components/document-header/document-header.scss

```diff
@@ -1,3 +1,5 @@
+@use "../grid/grid.variables.scss" as *;
```

Comment:
willen we toch maar wel meteen de namespacing goed zetten? `as *` -> `as grid-variables`

Reply:
Het moest zelfs, de build viel hierover 🤷